### PR TITLE
TRSv2-530 - Create account: Check with user if organisation already exists

### PR DESCRIPTION
**Pull request by nahmed99**

### CODE REVIEW

File: trade_remedies_public/templates/v2/registration/how_to_get_account.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{% extends "v2/govuk/base_with_form.html" %}
```
Comment (chris-pettinga) [inline trade_remedies_public/templates/v2/registration/how_to_get_account.html:1]: no need to extend from base_with_form, can just use base.html instead
```suggestion
{% extends "v2/govuk/base.html" %}
```

File: trade_remedies_public/templates/v2/registration/how_to_get_account.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+{% block page_title %}
+    How to get an account
+{% endblock %}
+
+{% block heading %}
+    How to get an account
+{% endblock heading %}
+
+{% block form_content %}
```
Comment (chris-pettinga) [inline trade_remedies_public/templates/v2/registration/how_to_get_account.html:12]: Then here would be row_content
```suggestion
{% block row_content %}
```

File: trade_remedies_public/templates/v2/registration/registration_check_existing_organisation.html [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{% extends "v2/govuk/base_with_form.html" %}
+{% load static %}
+
+{% block page_title %}
+    Existing account for your organisation?
+{% endblock %}
+
+{% block form_content %}
+    {% include 'v2/component_macros/two_radio_buttons.html' with value=radio_value value_one="yes" value_two="no" label_size='xl' label="Has anyone already set up an account for your organisation?" id_one="direct_employee" label_one="Yes" id_two="representative" label_two="No" name="organisation_already_setup" inline="y" %}
```
Comment (chris-pettinga) [inline trade_remedies_public/templates/v2/registration/registration_check_existing_organisation.html:9]: inline should be True so it's clearer to read
```suggestion
    {% include 'v2/component_macros/two_radio_buttons.html' with value=radio_value value_one="yes" value_two="no" label_size='xl' label="Has anyone already set up an account for your organisation?" id_one="direct_employee" label_one="Yes" id_two="representative" label_two="No" name="organisation_already_setup" inline=True %}
```

File: trade_remedies_public/registration/views/views.py [hunk 4]
```diff
@@ -437,6 +438,30 @@ def get_next_url(self, form=None):
...
+):
+    template_name = "v2/registration/registration_check_existing_organisation.html"
+    form_class = V2RegistrationViewConfirmExistingOrganisationForm
+
+    def get(self, request, *args, **kwargs):
+        self.reset_session(request)
+        return super().get(request, *args, **kwargs)
+
+    def form_valid(self, form):
+        self.update_session(self.request, form.cleaned_data)
```
Comment (chris-pettinga) [inline trade_remedies_public/registration/views/views.py:452]: No need to update session I don't think

File: trade_remedies_public/config/urls.py [hunk 5]
```diff
@@ -254,6 +254,16 @@
     path(
         "companieshouse/search/", core_views.CompaniesHouseSearch.as_view(), name="companieshouse"
     ),
+    path(
+        "register/confirm_existing_org",
+        register_views.V2RegistrationViewConfirmExistingOrganisation.as_view(),
+        name="v2_confirm_existing_org",
+    ),
+    path(
```
Comment (chris-pettinga) [inline trade_remedies_public/config/urls.py:262]: you don't need to create an entire view just for one template. You can render the template in urls.py technically:

`path('v2_how_to_get_account/',TemplateView.as_view(template_name="v2/registration/how_to_get_account.html"))`

File: trade_remedies_public/templates/v2/registration/registration_check_existing_organisation.html (hunk 3 above)
Comment (nahmed99) [reply trade_remedies_public/templates/v2/registration/registration_check_existing_organisation.html:9]: Yes. I tried that, it didn't work - must have used lower case "t".

File: trade_remedies_public/templates/v2/registration/registration_check_existing_organisation.html [hunk 6]
```diff
@@ -0,0 +1,11 @@
+{% extends "v2/govuk/base_with_form.html" %}
+{% load static %}
+
+{% block page_title %}
+    Existing account for your organisation?
+{% endblock %}
+
+{% block form_content %}
+    {% include 'v2/component_macros/two_radio_buttons.html' with value=radio_value value_one="yes" value_two="no" label_size='xl' label="Has anyone already set up an account for your organisation?" id_one="direct_employee" label_one="Yes" id_two="representative" label_two="No" name="organisation_already_setup" inline=True %}
```
Comment (yusufsheiqh) [inline trade_remedies_public/templates/v2/registration/registration_check_existing_organisation.html:9]: We will need to pull and merge the latest changes from `develop`, as all pages must have a heading tag.

```suggestion
    {% include 'v2/component_macros/two_radio_buttons.html' with value=radio_value value_one="yes" value_two="no" label_size='xl' label="Has anyone already set up an account for your organisation?" id_one="direct_employee" label_one="Yes" id_two="representative" label_two="No" name="organisation_already_setup" inline=True heading=True %}
```

Comment (nahmed99) [reply trade_remedies_public/templates/v2/registration/registration_check_existing_organisation.html:9]: Heading tag in the radio buttons macro - that was added yesterday? Okay.

Comment (yusufsheiqh) [reply trade_remedies_public/templates/v2/registration/registration_check_existing_organisation.html]: Resolved in https://github.com/uktrade/trade-remedies-public/pull/298/commits/059dd88e1cb7ea8908b2dd0d15f17d0242b198ea.

**Review by chris-pettinga — APPROVED** [review]

**Pull request merged**